src/defaults/html.pipe.js

```diff
@@ -71,6 +72,7 @@ const htmlpipe = (cont, context, action) => {
     .use(fetch).expose('fetch').when(hascontent)
     .use(parse).expose('parse')
     .use(parseFrontmatter)
+    .use(clearPositions)
```

Comment:
The conditional should be here.

Reply:
```suggestion
    .use(clearPositions).when(production)
```

would be a simple fix, re-using the "are we in production or debug mode" check that already exists.

Reply:
why not before `parseFrontmatter` ?